templates/accounts/signup.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'base.html' %}
+
+{% block title %}Sign Up | Solved Chinese{% endblock title %}
+
+{% block content %}
+  <a href="{% url student_signup %}" class="button button-primary">
```

Comment:
it should be `{% url 'student_signup' %}` because the reverse url should be a string